src/main/java/com/guysfromusa/carsgame/entities/MovementsHistoryEntity.java

```diff
@@ -0,0 +1,75 @@
+package com.guysfromusa.carsgame.entities;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.PrePersist;
+import java.time.LocalDateTime;
+
+import static java.time.LocalDateTime.now;
+
+/**
+ * Created by Dominik Zurek, 26.02.18
+ */
+
+@Entity(name = "MOVEMENT_HISTORY")
```

Comment:
just Movement?
@Entity(name=movement)
@Table(name=movement)

Reply:
I think is better is movement_history because we use this enttity to show hiostory ...